In [11]:
import pandas as pd

#Create a Data Frame
dataFrame = pd.read_csv('AggregateIncome.csv')

#Turn the strings in the income column into numerics
dataFrame['estimate__aggregate_household_income_in_the_past_12_months__'] = pd.to_numeric(dataFrame['estimate__aggregate_household_income_in_the_past_12_months__'], errors='coerce')

#Sort income decending from greatest to least
sortedIncome = dataFrame.sort_values(by='estimate__aggregate_household_income_in_the_past_12_months__', ascending=False)

# Display the top 10 neighborhoods in terms of aggregate income
print(sortedIncome[['estimate__aggregate_household_income_in_the_past_12_months__','Neighborhood']].head(10))


    estimate__aggregate_household_income_in_the_past_12_months__  \
15                                        659886300.0              
20                                        605635700.0              
23                                        577467500.0              
44                                        366945900.0              
50                                        314958500.0              
55                                        267100700.0              
58                                        249898900.0              
59                                        242405400.0              
60                                        241377900.0              
63                                        233098400.0              

           Neighborhood  
15  Squirrel Hill South  
20            Shadyside  
23  Squirrel Hill North  
44            Brookline  
50         Point Breeze  
55     Mount Washington  
58           Bloomfield  
59        Highland Park  
60           Gree

## Top 10 Neighborhoods in Total Income
1. Squirell Hill South - 659,886,300
2. Shadyside - 605,635,700
3. Squirrel Hill North - 577,467,500
4. Brookline - 366,945,900
5. Point Breeze - 314,958,500
6. Mount Washington - 267,100,700
7. Bloomfield - 249,898,900
8. Highland Park - 242,405,400
9. Greenfield - 241,377,900
10. South Side Flats - 233,098,400

In [7]:
#Create a Data Frame
dataFrame = pd.read_csv('Neighborhoods_with_SNAP_Data.csv')

#Remove the dollar sign and convert type to float
dataFrame['SNAP_All_csv_Median_Home_Value_'] = dataFrame['SNAP_All_csv_Median_Home_Value_'].replace(r'[\$,]', '', regex=True).astype(float)

#Sort income decending from greatest to least
sortedIncome = dataFrame.sort_values(by='SNAP_All_csv_Median_Home_Value_', ascending=False)

# Display the top 10 neighborhoods in terms of median home value
print(sortedIncome[['SNAP_All_csv_Median_Home_Value_','Neighborhood_2010_HOOD']].head(10))

    SNAP_All_csv_Median_Home_Value_ Neighborhood_2010_HOOD
65                         254667.0    Squirrel Hill North
1                          222423.0          North Oakland
4                          215900.0         Strip District
34                         164000.0           Point Breeze
55                         159700.0         Allegheny West
57                         153560.0              Shadyside
25                         134633.0    Squirrel Hill South
49                         115600.0             Friendship
68                         113000.0          Regent Square
37                          99900.0          Highland Park


## Top 10 Neighborhoods by Median Home Value
1. Squirrel Hill North - 254,667
2. North Oakland - 222,423
3. Strip District - 215,900
4. Point Breeze - 164,000
5. Allegheny West - 159,700
6. Shadyside - 153,560
7. Squirrel Hill South - 134,633
8. Friendship - 115,600
9. Regent Square - 113,000
10. Highland Park - 99,900

Each of these submetrics dictate the amount of wealth in the neighborhoods, which is an important thing to consider when picking a place to live. Low-income directly translates to crime, so you want to pick a place that is generally wealthy. This is important not only from a crime standpoint, but if also leads to better overall cleanliness, more shopping and food options, etc.